lib/matplotlib/axes/_base.py

```diff
@@ -2632,6 +2618,8 @@ def set_frame_on(self, b):
         Parameters
         ----------
         b : bool
+            ..
+                ACCEPTS: bool
```

Comment:
Why did you add this line?

Reply:
... because the PR dates from before we got rid of them :)
Removing it.